File: exo_AND.py
x = input("météo ? ")
y = input("température ? ")

if x == "raining" and y == "cold":
    print("take an umbrella and a warm jacket.")
elif x == "raining" and y == "warm":
    print("take an umbrella and a t-shirt.")
elif x == "sunny" and y == "cold":
      print("take sunglasses and a warm jacket.")
elif x == "sunny" and y == "warm":
      print("take sunglasses and a t-shirt.")
else: print("stay home!")

File: exo_AND.js

var temperature = prompt("Quel température fait-il ?");
var météo = prompt("Quel météo ?");

if (temperature == "cold" && météo == "raining"){
    console.log("take an umbrella and a warm jacket.");
}else if (temperature == "warm" && météo == "raining"){
    console.log("take an umbrella and a t-shirt.");
}else if (temperature == "cold" && météo == "sunny"){
    console.log("take sunglasses and a warm jacket.");
}else if (temperature == "warm" && météo == "sunny"){
    console.log("take sunglasses and a t-shirt.");
 }else{
    "stay home!";
 }

